bundles/org.openhab.core.ephemeris/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.core.bundles</groupId>
+    <artifactId>org.openhab.core.reactor.bundles</artifactId>
+    <version>2.5.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.core.ephemeris</artifactId>
+
+  <name>openHAB Core :: Bundles :: Ephemeris</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.openhab.core.bundles</groupId>
+      <artifactId>org.openhab.core.config.core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
```

Comment:
Please do not add any external dependency here.
That should be injected by the compile BOM already.

Reply:
That's what I expected as well. But as you can see [here](https://github.com/openhab/openhab-core/pull/506/commits), the PR build failed without it and once I added it, it succeeded. Any idea why?

Reply:
On my machine it is working (building the project after removing that lines).

Looking at the check of the first commit:
* continuous-integration/travis-ci/pr: success
* PR-openHAB-Core: failure

I assume you should check your Jenkins configuration.

Reply:
IMHO my comment is still valid and should be addressed.

Reply:
Definitely - sorry, it slipped through. Fixed now.